azkaban-common/src/main/java/azkaban/event/Event.java

```diff
@@ -71,4 +84,15 @@ public static Event create(Object runner, Type type, EventData eventData) throws
     return new Event(runner, type, eventData);
   }
 
+  /**
+   * Creates a new event.
+   *
+   * @param runner runner.
+   * @param type type.
+   * @return New Event instance.
+   * @throws NullPointerException if EventData is null.
+   */
+  public static Event create(Object runner, Type type) {
```

Comment:
Both the constructor and the static create method are very similar. What's the advantage of having a private constructor here?